Makefile

```diff
@@ -99,15 +101,23 @@ dev-setup:
 	@echo "------------------"
 
 run-x86:
-	@docker-compose -f ./deploy/docker/clickhouse-setup/docker-compose.yaml up -d
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.yaml up -d
 
 run-arm:
-	@docker-compose -f ./deploy/docker/clickhouse-setup/docker-compose.arm.yaml up -d
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.arm.yaml up -d
+
+down-x86:
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.yaml down -v
+
+down-arm:
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.arm.yaml down -v
 
 clear-standalone-data:
-	@docker run --rm -v "$(PWD)/deploy/docker/clickhouse-setup:/pwd" busybox \
-	sh -c "cd /pwd/data && rm -rf alertmanager/* clickhouse/* signoz/*"
+	@cd $(STANDALONE_DIRECTORY)
+	@docker run --rm -v "data:/pwd" busybox \
+	sh -c "cd /pwd && rm -rf alertmanager/* clickhouse/* signoz/*"
```

Comment:
@prashant-shahi can't we just delete `STANDALONE_DIRECTORY/data` folder rather than mounting that path to a container and deleting the same data?

Reply:
If not this solution, we will need `sudo` permissions to delete from the CLI as _docker_ is the owner for those files.
Moreover, we would want the 1 level below folders inside data intact. We just need to clear everything from those folders.

In Docker Standalone, it will most likely not cause any issue.
However in Docker Swarm, it is mandatory to have those folders when bringing up the cluster.

Reply:
ok